### PR TITLE
Deploy with Travis

### PR DESCRIPTION
**Pull request by JayH5**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,16 +1,36 @@
...
+    - secure: "W21cHV4ow3eGFZJmI5dpTs9zGIjRcrYkMbwE//TS8TJP1zNm6/kcRauyapyxo+hjlf717HH8mM/N8NiJ96w3q440s5QI28I7UpgJH/9Xt2czVcCC/dDa5sOsYqZX6phTc2HulPW8hLCaW4YK2cxe3sSxq2RqnHj42hlul7Nw68SYoKlScWUwX0tIRpUOQWPFpv1hFwm3yYZAEJ2OLma0yd+pvW2QlYe0C9KCAEhSXTn+PErvnkACzP2XkpFyleQ41Y+0E459XQRYsG+KvgMW8UcVPZO4zoeQ3lyAKPNtbQ3GQO0U79B+AQ3bGDleGpwDSzckGa9xulL5vIAS8RtwUPYP0VWhBj8ST7lKNh81F/5twWD8Z8mQp2ne12sGb1J0WCvpfVlJq5IgqriF04AHhC5SBzQDg+fqss8NEPzqG0nKBmPeCaKiuu5CTw+XI/O8bRnmxS2BiPNoepkpBMFLvlrVJuiFlZgUyljM9CNEZ7XvkphcULyjICUPVEyJE5vS3p/49HoctJIU3OOyohj0oDNYJd5pSn1DuXd3GQKOiUwdFTqoILxrtnPkcEiX79InsVk6yAQYrFAHN0Qi8DuQqgKRmJM6SNpzxWJxKvRJqajpPhCyOY/IdRyjyNYNd5GSconh6btRHU0fUwACaYEgzcslushzLlsuxaOPU2ppkC0="
 
 # Update Docker Engine
 before_install:
   - sudo apt-get update
   - sudo apt-get install -qy -o Dpkg::Options::="--force-confold" docker-engine
+install: []
+
+before_script:
+  - version="$(awk '$2 == "FREESWITCH_VERSION" { print $3; exit }' Dockerfile | cut -d'~' -f1)"
```
Comment (bearnard) [inline .travis.yml:19]: is this enough to handle other image changes (i.e not related to the version of freeswitch) eg updated/new modules?

Comment (JayH5) [reply .travis.yml:19]: No it isn't. But I'm not sure where we'd put that in the version information.

Comment (bearnard) [reply .travis.yml:19]: Yeah, I think it's fine with a low frequency release image like this.

Also that leading double quote expected? 
```
$ awk '$2 == "FREESWITCH_VERSION" { print $3; exit }' Dockerfile | cut -d'~' -f1
"1.6.15

```

Comment (JayH5) [reply .travis.yml:19]: Doesn't do that for me :(

Comment (JayH5) [reply .travis.yml:19]: @bearnard I think you may be testing against the wrong branch... I removed the `"` around the `ENV` value in #7 recently.

Comment (bearnard) [reply .travis.yml:19]: Ah cool I suspected that.

**Comment by bearnard** [comment]
👍

**Pull request merged**